service/init.go

```diff
@@ -669,7 +675,9 @@ func (amf *AMF) UpdateConfig(commChannel chan *protos.NetworkSliceResponse) bool
 }
 
 func (amf *AMF) SendNFProfileUpdateToNrf() {
-	for rocUpdateConfig := range RocUpdateConfigChannel {
+	//for rocUpdateConfig := range RocUpdateConfigChannel {
+	for {
+		rocUpdateConfig := <-RocUpdateConfigChannel
```

Comment:
was this done to avoid closing channels cause range needs us to close the channel ?